package.json

```diff
@@ -44,12 +44,14 @@
     "preview": "vite preview",
     "api:build": "cd api && yarn build",
     "build:vercel": "NODE_OPTIONS=--max_old_space_size=5000 vite build",
-    "format": "prettier . --write"
+    "format": "prettier . --write",
+    "lint": "eslint src",
+    "prepare": "husky"
   },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
+  "lint-staged": {
+    "*.{js,jsx,ts,tsx}": [
+      "yarn prettier --write",
```

Comment:
should we run prettier later?
I think there are eslint rules that may add/remove code, in the main repo we have first eslint and then prettier

Reply:
Fixed in https://github.com/metabase/shoppy/pull/48/commits/6db3e92dda5dd90606781d10acad023876666a7b